ocs_authentication/auth_profile/migrations/0001_initial.py

```diff
@@ -17,10 +17,9 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='AuthProfile',
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('staff_view', models.BooleanField(default=False)),
-                ('access_token', models.CharField(default='', max_length=255)),
-                ('refresh_token', models.CharField(default='', max_length=255)),
+                ('api_token', models.CharField(default='', max_length=255)),
```

Comment:
Just trying to understand better - what will the migration look like for the client applications' DBs? Since we are removing the access/refresh tokens from the AuthProfile and replacing them with a single `api_token`, how is the `api_token` populated for existing users?

Reply:
So this library is new / hasn't been deployed in use anywhere, and this AuthProfile model is new to the library, so no client's DBs should be affected if that makes sense. Elisabeth originally planned on storing the access/refresh tokens in here, but since we moved away from that I removed them and switched to the `api_token`, which is the OAuth Server's api_token. It is being stored in here rather than just using the DRF internal AuthToken so that we can support both for a while before moving to just the api_token, although I'm not sure if anyone is using the current DRF api tokens on anything other than the Observation Portal. If you look at the Observation Portal PR, I have a management command that will be called to populate initial values for the users and their AuthProfiles with `api_token`. After that is seeded once, it will be kept up to date through the OAuth Server (Obs Portal) updating the clients on any user profile or token change.